vms/rpcchainvm/vm_server.go

```diff
@@ -106,10 +106,8 @@ func (vm *VMServer) Initialize(ctx context.Context, req *vmpb.InitializeRequest)
 	if err != nil {
 		return nil, err
 	}
-	nodeID, err := ids.ToNodeID(req.NodeId)
-	if err != nil {
-		return nil, err
-	}
+	nodeID := ids.NodeIDFromBytes(req.NodeId, len(req.NodeId))
```

Comment:
This should error if the bytes aren't 32 or 20

Reply:
Done